### ISSUE TITLE
MobHealth similar function.

### ISSUE THREAD

**Issue by TheIntelloBox** [Rocologo/MobHunting#401]
Hello,

https://dev.bukkit.org/projects/mobhealth

This plugin is very good to show the health of the mobs and the players, so it would be great if you put a similar function in MobHunting.

Thanks ! :)

**Comment by Rocologo** [comment]
Why should I add something which works in another plugin???? :-)

**Comment by TheIntelloBox** [comment]
Because the plugin doesn't work. xD

**Comment by MCTyler** [comment]
We can Update that Plugin. I'm sure its just a few simple fixes since it's already 1.10.

**Comment by TheIntelloBox** [comment]
Yeah but I'm noob in Java so I don't know how to do it, I need someone that can do it...

**Comment by MCTyler** [comment]
Yea It appears that somer dependencies are not even available any more...Sry. I hate Maven.

**Comment by MCTyler** [comment]
Heres the ones I have found so far....
![screenshot from 2018-05-06 15-09-54](https://user-images.githubusercontent.com/6390898/39677380-9d99a7a6-513f-11e8-8c1d-abdf735aed1e.png)

**Comment by TheIntelloBox** [comment]
Oh, do you think you can recode it ? Or Rocologo ?

**Comment by MCTyler** [comment]
I can't. Maybe you can recruit a Dev on Spigot for a small fee.

**Comment by Rocologo** [comment]
I use this one on my own server. It works fine with MC 1.12.2

**Comment by MCTyler** [comment]
Maybe Op downloaded a old copy. I don't use it.

**Comment by Rocologo** [comment]
Sorry I forgot the link 

https://www.spigotmc.org/resources/action-bar-health.2661/
